features/support/env.rb

```diff
@@ -20,13 +20,24 @@ def launch_packager
   run_script('features/scripts/launch_packager.sh')
 end
 
+def set_envfile(envfile)
+  Dir.chdir('features/fixtures/sampler') do
+    FileUtils.rm('.env')
+    FileUtils.cp("scenario_envs/#{envfile}.env", '.env')
```

Comment:
We should add error handling to all uses of FileUtils so that an error is logged if mazerunner fails to perform a file operation - otherwise it'll be hard to debug scenarios.